File: app/src/main/java/workshop/akbolatss/tagsnews/screen/news/NewsView.java
package workshop.akbolatss.tagsnews.screen.news;

import me.toptas.rssconverter.RssFeed;
import workshop.akbolatss.tagsnews.base.BaseView;
import workshop.akbolatss.tagsnews.base.ErrorView;
import workshop.akbolatss.tagsnews.base.LoadingView;


public interface NewsView extends BaseView, LoadingView, ErrorView {

    public void onLoadNews(RssFeed rssFeed);

}

File: app/src/main/java/workshop/akbolatss/tagsnews/screen/news/NewsView.kt
package workshop.akbolatss.tagsnews.screen.news

import me.toptas.rssconverter.RssFeed
import workshop.akbolatss.tagsnews.base.BaseView
import workshop.akbolatss.tagsnews.base.ErrorView
import workshop.akbolatss.tagsnews.base.LoadingView

/**
 * MVP View for #NewsView
 * @see NewsFragment
 */
interface NewsView : BaseView, LoadingView, ErrorView {

    fun onLoadNews(rssFeed: RssFeed)
}
